marketplace-solver/src/database.rs

```diff
@@ -72,6 +73,17 @@ impl PostgresClient {
 
         let connection = options.connect(postgres_url.as_str()).await?;
 
+        if reset {
+            sqlx::query("DROP SCHEMA public CASCADE;")
+                .execute(&connection)
+                .await
+                .unwrap();
+            sqlx::query("CREATE SCHEMA public;")
+                .execute(&connection)
+                .await
+                .unwrap();
+        }
```

Comment:
Previous comments by Jeb and Tim were that we should exit if we do the reset. Currently it looks like we don't do that, but I could be wrong.

Reply:
That was for the sequencer binary. In the utils binary, we just call connect(), which resets and returns a client, but we don't do anything after that.